source/2020-05-29-the-ember-times-issue-150.md

```diff
@@ -14,14 +14,28 @@ READMORE
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## ["My Experience with Ember.js" Part 2 🎬](https://www.youtube.com/watch?v=R2JdP4lb5Xw)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Cal Woolgar (@callam_woolgar)](https://twitter.com/callam_woolgar) has a second short video out with his new series, [Creating an Ember Application!](https://www.youtube.com/watch?v=R2JdP4lb5Xw) He gets into a little more detail in this video and demonstrates how easy it is to install and get started with Ember.js and it's cli.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+```bash
+npm install -g ember-cli
+```
+
+This video touches on an [Ember app's anatomy](https://guides.emberjs.com/release/getting-started/anatomy-of-an-ember-app/) and how to create a basic [Ember.js](https://emberjs.com/) app using the ember cli for a fully working application with **one line of code!** 🎉
```

Comment:
I think this second part of this sentence, starting with "how to create," is a bit too long.

It seemed to me like you wanted to emphasize an Ember app's well-defined project structure and how a reader can start with a single-line command.

What do you think about something like:
```suggestion
To create an Ember app with default [project structure](https://guides.emberjs.com/release/getting-started/anatomy-of-an-ember-app/):
```